### PR TITLE
Add more jank tracking

### PR DESCRIPTION
**Pull request by keyboardsurfer**

### CODE REVIEW

File: feature/author/src/main/java/com/google/samples/apps/nowinandroid/feature/author/AuthorScreen.kt [hunk 1]
```diff
@@ -91,6 +93,7 @@ internal fun AuthorScreen(
     onBookmarkChanged: (String, Boolean) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+    TrackScrollJank(scrollableState = rememberLazyListState(), stateName = "author:column")
```
Comment (tunjid) [inline feature/author/src/main/java/com/google/samples/apps/nowinandroid/feature/author/AuthorScreen.kt:96]: Should the `LazyListState` passed to `TrackScrollJank` also be passed to the `LazyColumn`?

File: feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt [hunk 2]
```diff
@@ -175,10 +175,7 @@ fun ForYouScreen(
             }
         }
 
-        val tag = "forYou:feed"
-
-        val lazyGridState = rememberLazyGridState()
-        TrackScrollJank(scrollableState = lazyGridState, stateName = tag)
+        TrackScrollJank(scrollableState = rememberLazyGridState(), stateName = "forYou:feed")
```
Comment (tunjid) [inline feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt:178]: I'm not too sure how tracking Jank works, but is it an error to pass the same instance of `LazyGridState` to `TrackScrollJank` and `LazyVerticalGrid`? It seems the stats tracked by `TrackScrollJank` won't match the scroll behavior of the Composable.

**Review by tunjid — APPROVED** [review]
LGTM!

File: feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt [hunk 3]
```diff
@@ -86,6 +88,8 @@ fun InterestsScreen(
     switchTab: (Int) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+
+    TrackScrollJank(scrollableState = rememberLazyListState(), stateName = "Interests:screen")
```
Comment (dturner) [inline feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt:92]: For consistency: 
```suggestion
    TrackScrollJank(scrollableState = rememberLazyListState(), stateName = "interests:screen")
```

**Review by dturner — APPROVED** [review]
One very minor nit

File: feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt [hunk 4]
```diff
@@ -86,6 +88,8 @@ fun InterestsScreen(
     switchTab: (Int) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+
+    TrackScrollJank(scrollableState = rememberLazyListState(), stateName = "interests:screen")
```
Comment (mlykotom) [inline feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt:92]:
```suggestion
    val state = rememberLazyListState()
    TrackScrollJank(scrollableState = state, stateName = "interests:screen")
```

Comment (mlykotom) [reply feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt:92]: You need to pass the `state` to the lazyList that it refers to .. Actually what lazyList does it refer to? 🤔 
Maybe this should be part of the `InterestsContent` composable?

File: feature/topic/src/main/java/com/google/samples/apps/nowinandroid/feature/topic/TopicScreen.kt [hunk 5]
```diff
@@ -90,6 +92,7 @@ internal fun TopicScreen(
     onBookmarkChanged: (String, Boolean) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+    TrackScrollJank(scrollableState = rememberLazyListState(), stateName = "topic:screen")
     LazyColumn(
         modifier = modifier,
```
Comment (mlykotom) [inline feature/topic/src/main/java/com/google/samples/apps/nowinandroid/feature/topic/TopicScreen.kt:97]:
```suggestion
    val state = rememberLazyListState()
    TrackScrollJank(scrollableState = state, stateName = "topic:screen")
    LazyColumn(
        state = state,
        modifier = modifier,
```

**Review by mlykotom — COMMENTED** [review]
Pls verify you use the same state for tracking and for the actual Lazy list/grid.

File: feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt (hunk 4 above)
Comment (keyboardsurfer) [reply feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt]: Removed this again.

**Pull request merged**